adafruit_platformdetect/constants/boards.py

```diff
@@ -165,6 +165,7 @@
 
 RADXA_ZERO = "RADXA_ZERO"
 RADXA_CM3 = "RADXA_CM3"
+RADXA_3A = "ROCK_PI_3A"
```

Comment:
Could we change this to ROCK_PI_3A to match the other ROCK PI boards?

Reply:
Ok, will change it also downstream (blinka)